src/bin-install.ts

```diff
@@ -14,15 +14,21 @@ typings install [<name>=]<location>
   <location>  The location to read from (described below)
 
 Valid Locations:
-  [<source>!]<pkg>[@<version>][#<tag>]
+  <pkg>[@<version>][#<tag>]
+    Install from "defaultSource" specified in `.typingsrc`
+      default to "npm" for non-ambient package
+      default to "dt" for ambient package
+  <source>!<pkg>[@<version>][#<tag>]
+    Install from registry mirror or DefinitelyTyped (see <source> below)
```

Comment:
I'd like to avoid calling out DefinitelyTyped so explicitly, I still don't see the reason why. It's just making a bigger mess to clean up later.